h/services/developer_token.py

```diff
@@ -42,7 +42,7 @@ def create(self, userid):
         """
         user = self.user_svc.fetch(userid)
         token = models.Token(
-            userid=user.userid, user_id=user.id, value=self._generate_token()
+            user=user, userid=user.userid, value=self._generate_token()
```

Comment:
Writing both `user` (which writes `user_id`) and the old `userid` (this will be removed soon).